libs/wire-api/src/Wire/API/Error/Galley.hs

```diff
@@ -162,6 +160,59 @@ type family MissingPermissionMessage (a :: Maybe Perm) :: Symbol where
 instance APIError GalleyError where
   toWai err = toWai $ case err of
     InvalidAction -> dynError @(MapError 'InvalidAction)
+    InvalidTargetAccess -> dynError @(MapError 'InvalidTargetAccess)
```

Comment:
This whole instance feels like something that should be TH if we can't derive it.